library/odoo

```diff
@@ -1,6 +1,6 @@
 Maintainers: Christophe Monniez <moc@odoo.com> (@d-fence)
 GitRepo: https://github.com/odoo/docker
-GitCommit: d04d885eea6b8f60ada727fcc498f5a8fd44da51
+GitCommit: 8a93c61c028c306c32610cdef4cfcb606edd4ff0
 
 Tags: 16.0, 16, latest
 Architectures: amd64
```

Comment:
We don't yet use `ARG TARGETARCH` in official-images. We recommend using `apk --print-arch` or `dpkg --print-architecture` as the most reliable options to retrieve the userspace architecture of the image.

You'll also need to expand this `Architectures:` value in the library file for the image tags that support building on other architectures:

```diff
-Architectures: amd64
+Architectures: amd64, arm64v8, ppc64le
```

Reply:
Hello. I changed to use `dpkg --print-architecture` when TARGETARCH is not set.
I think that it should work in both cases.

Reply:
`TARGETARCH` isn't interchangeable with `dpkg --print-architecture`. The `dpkg` values convey more information when run on some architectures (32bit arm architectures for example).

Reply:
I agree but I think that it should work as we only want to set `TARGETARCH` when it's not already set and we only build for `amd64, arm64v8, ppc64le`. I tested with buildx and it worked as expected.